backend/src/main/java/com/google/sps/datastore/ParticipantDatastore.java

```diff
@@ -153,8 +154,10 @@ public Participant getParticipantFromUsername(String username) {
 
     // Convert entities to list of participants
     List<Participant> participants = new ArrayList<Participant>();
-    for (Entity entity : results.asIterable()) {
-      participants.add(getParticipantFromEntity(entity));
+    if (results != null) {
```

Comment:
when is results null? when there are no entries found?

Reply:
Oops, I deployed the backend and was debugging and forgot to remove this. Removed again.